src/us/kbase/shock/client/ShockFileInformation.java

```diff
@@ -23,7 +23,6 @@ private ShockFileInformation(){}
 	@JsonProperty("checksum")
 	private Map<String, String> checksum;
```

Comment:
do we need @JsonProperty("checksum") here?
Is not mapping to a diff name though.

Reply:
Probably not, but if I fix all the weird problems with this code this PR will explode, so we should limit the scope to problems with the PR specifically. 

Really there's not much need for this client any more other than backwards compatibility, it's easy enough to do most operations via a generic REST client. I don't see much of a future for it long term, but for now we need it